README.md

```diff
@@ -17,9 +21,9 @@ Shared scripts for grumbler based modules.
     "test": "npm run lint && npm run flow-typed && npm run flow && npm run karma",
     "build": "npm run test && npm run babel && npm run webpack && npm run flow:build",
     "release": "npm run release:patch",
-    "release:patch": "./node_modules/grumbler-scripts/publish.sh patch",
-    "release:minor": "./node_modules/grumbler-scripts/publish.sh minor",
-    "release:major": "./node_modules/grumbler-scripts/publish.sh major",
+    "release:patch": "./node_modules/@krakenjs/grumbler-scripts/publish.sh patch",
```

Comment:
Do we need to keep the `publish.sh` file in order to support these? Or can we delete these too?

Reply:
i actually just removed these but awesome catch!